obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: 'Build SRPM'
+  command: "tito build --srpm --offline"
+  args:
+    chdir: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}"
+  register: srpm_build
+
+- debug:
+    msg: "{{ srpm_build.stdout_lines }}"
```

Comment:
I'd use `stdout` instead of `stdout_lines` here?

```
stdout

Creating output directory: /tmp/tito
Building package [1.2-3]

Wrote: /tmp/tito/bleh-1.2-3.fc28.src.rpm
```

```
stdout_lines
[u'Creating output directory: /tmp/tito', u'Building package [1.2-3]', u'', u'Wrote: /tmp/tito/bleh-1.2-3.fc28.src.rpm']
```